core/src/main/java/feign/MethodMetadata.java

```diff
@@ -118,6 +119,15 @@ public MethodMetadata queryMapEncoded(boolean queryMapEncoded) {
     return this;
   }
 
+  public boolean alwaysEncodeBody() {
+    return alwaysEncodeBody;
+  }
+
+  MethodMetadata alwaysEncodeBody(boolean alwaysEncodeBody) {
```

Comment:
Please annotate both methods with `@Experimental`, just in case we change our minds later on.

Reply:
Ok, I will do that. Thanks.

Reply:
Done